wbooks/src/app/components/BookList/styles.ts

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  list: {
+    width: '90%'
```

Comment:
Use `marginHorizontal` instead of define fixed widths => `marginHorizontal: 20`

Reply:
That's a pretty good approach. I'll update it.